app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.java

```diff
@@ -153,13 +154,28 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         }
 
         Engagements.initUserEngagement(this);
+
+        Intent intent = getIntent();
+        if (intent != null) {
```

Comment:
How could `getIntent()` return `null` here?

Reply:
Ups. Fixed in d1ceac302cd88e52d3e1c937b01e2c489d6e32d6.